android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -471,6 +477,28 @@ private static void setInboxStyle(NotificationDetails notificationDetails, Notif
         builder.setStyle(inboxStyle);
     }
 
+    private static void setMediaStyle(NotificationDetails notificationDetails, NotificationCompat.Builder builder) {
+        MediaStyleInformation mediaStyleInformation = (MediaStyleInformation) notificationDetails.styleInformation;
+        int[] showActionsInCompactView = convertIntegers(mediaStyleInformation.showActionsInCompactView);
+
+        androidx.media.app.NotificationCompat.MediaStyle mediaStyle = new androidx.media.app.NotificationCompat.MediaStyle();
+        mediaStyle.setShowActionsInCompactView(showActionsInCompactView);
```

Comment:
From my own testing and looking at the doc that has a sample code snippet (see https://developer.android.com/reference/androidx/media/app/NotificationCompat.MediaStyle#summary), specifying actions doesn't appear to be needed for the media style notification to work. With that being the case, can you remove the code relating to the actions in both the Java and Dart side?

Reply:
Yes, it is correct that it is not needed. I added it because it will be a feature once the notification actions will be added to the plugin. Anyway I'll remove it.